cirq/google/merge_interactions.py

```diff
@@ -30,9 +30,11 @@ class MergeInteractions(circuits.PointOptimizer):
     def __init__(self,
                  tolerance: float = 1e-8,
                  allow_partial_czs: bool = True,
+                 decompose_partial_czs: bool = False,
                  extensions: Extensions = None) -> None:
         self.tolerance = tolerance
         self.allow_partial_czs = allow_partial_czs
+        self.decompose_partial_czs = decompose_partial_czs
```

Comment:
Why is this separate from the "allow_partial_czs" option? How do they differ?

Reply:
allow_partial_czs controls the output of decomposition.  decompose_partial_czs changes when a sequence of gates is decomposed.

Reply:
They could be merged into one flag.

Reply:
I think they should be merged into one flag.

Add a test that checks that MergeInteractions actually decomposes single partial CZs when the flag is set.

Reply:
Done.  I already have that test (test_decompose_partial_czs).